### PR TITLE
Makes space transition real cool

### PR DESCRIPTION
**Pull request by Razharas**
Makes space transition real cool
No more txt shite, only defines now
Fixes away missions derping

### CODE REVIEW

File: _maps/DiscStation.dm [hunk 1]
```diff
@@ -14,8 +14,10 @@ Names of z-level do not matter, but order does greatly, for instances such as ch
         #include "map_files\generic\z6.dmm"
         #include "map_files\generic\z7.dmm"
 
+
         #define MAP_FILE "DiscStation.dmm"
         #define MAP_NAME "DiscStation"
+        #define MAP_TRANSITION_CONFIG	list("Main Station" = 2,"CentComm" = 1,"Abandoned_Satellite" = 2,"Derelicted Station" = 2,"Mining Asteroid" = 2,"Empty Area 1" = 2,"Empty Area 2" = 2)
```
Comment (RemieRichards) [inline _maps/DiscStation.dm]: >strings

STRINGS!?!!?

**Comment by Mloc** [comment]
yay this is much better
I'd probably have made maps a datum and moved away from defines but this is just as good.

File: _maps/DiscStation.dm (hunk 1 above)
Comment (Mloc) [inline _maps/DiscStation.dm]: string are great

Comment (RemieRichards) [inline _maps/DiscStation.dm]: He added defines for it.
So he should define the strings too, because magic strings make me sad ;_;

Comment (Razharas) [inline _maps/DiscStation.dm]: Do you want me to put defines inside defines?
image

Comment (RemieRichards) [inline _maps/DiscStation.dm]: Yes

Comment (Miauw62) [inline _maps/DiscStation.dm]: Yes

**Comment by Shadowlight213** [comment]
So what do the numbers in the defines mean now? Since it's not linked/unlinked anymore.

**Comment by Dorsisdwarf** [comment]
But what do we call the new system? Spacehypercube?

File: _maps/DiscStation.dm (hunk 1 above)
Comment (duncathan) [inline _maps/DiscStation.dm]: Also you should make 1 and 2 defines

**Comment by JJRcop** [comment]
@Dorsisdwarf It looks like this is still space torus just no .txt madness

**Comment by Razharas** [comment]
2 - usual linked space like station or asteroid
1 - selflooped with no neigbours like centcomm
0 - not affected at all like away missions

**Pull request merged**